app/genres/page.tsx

```
@@ -2,146 +2,88 @@
 
 import { useState, useCallback } from "react"
 import { useRouter } from "next/navigation"
-import { ArrowLeft, Play, Pause, Music2 } from "lucide-react"
+import {
+  ArrowLeft, Play, Pause, Music2, Heart, Plus, Search,
```

ESLint

[error] Member 'Pause' of the import declaration should be sorted alphabetically.

(sort-imports)